Client/Frontend/Popup/AlertPopupView.swift

```diff
@@ -56,32 +56,54 @@ class AlertPopupView: PopupView {
     }
     
     func updateSubviews() {
+        titleLabel.adjustsFontSizeToFitWidth = false
+        messageLabel.adjustsFontSizeToFitWidth = false
+
+        _updateSubviews()
+        
+        let paddingHeight: CGFloat = padding * 3.0
+        let externalContentHeight: CGFloat = dialogButtons.count == 0 ? paddingHeight
```

Comment:
Removed line break, think line is still decent length (< 120 at least), especially after removing the unneeded `CGFloat`.